tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -684,3 +684,29 @@ TEXT_DRAW_TEST_P(RTLText, DrawRTLText) {
 }
 
 INSTANTIATE_TEST_CASE_P(RTLTextDrawing, RTLText, ::testing::ValuesIn(c_rotations));
+
+TEXT_DRAW_TEST_F(CTDrawing, CGPathTranslation, WhiteBackgroundTest<>) {
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    // Creates path with current rectangle
+    woc::unique_cf<CGMutablePathRef> path{ CGPathCreateMutable() };
+    CGPathAddRect(path.get(), nullptr, { { 25, 50 }, bounds.size });
+
+    CGAffineTransform textMatrix = CGContextGetTextMatrix(context);
+    CGContextSetTextMatrix(context, CGAffineTransformRotate(textMatrix, 45 * M_PI / 180.0));
+    CGContextScaleCTM(context, 1.1, 1.3);
```

Comment:
I wanted to guarantee that the path's origin would be properly affected by the transforms, and I don't think a basic test would give us any more coverage.